### PR TITLE
Collapsing some wiki widget in the Mobile wiki page content

### PR DESCRIPTION
**Pull request by OleanjiKingCode**
# Collapsing some wiki widget in the Mobile wiki page content

_Since our priority should be the getting the user to see the wiki content quickiy i collapsed the widgets above the content

## How should this be tested?
By opening up a wiki on mobile

1. Before   
image

2. Now 
image

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/511

### CODE REVIEW

**Comment by kesar** [comment]
i would just collapse the commit message. main pic is powerful to get context of the wiki, i would not remove that as first element

**Comment by OleanjiKingCode** [comment]
> i would just collapse the commit message. main pic is powerful to get context of the wiki, i would not remove that as first element





Sir what do you mean by collapsing?  i dont really get it

**Comment by OleanjiKingCode** [comment]
> i would just collapse the commit message. main pic is powerful to get context of the wiki, i would not remove that as first element





Sir what do you mean by collapsing?  i dont really get it

**Comment by kesar** [comment]
Captura de Pantalla 2022-07-20 a las 18 46 38

this by default only in mobile

**Comment by kesar** [comment]
there are a bunch of widgets collapsed on desktop and mobile that were not in the specs.

The specs mentioned the commit msg. Some like twitter or related widgets sounds like a good idea to be collapsed by default on mobile, so feel free to keep it, but no reason for that on desktop since we have a bunch of space to fill on the screen 👍🏻

**Comment by OleanjiKingCode** [comment]
> there are a bunch of widgets collapsed on desktop and mobile that were not in the specs.
> 
> The specs mentioned the commit msg. Some like twitter or related widgets sounds like a good idea to be collapsed by default on mobile, so feel free to keep it, but no reason for that on desktop since we have a bunch of space to fill on the screen 👍🏻

yes sir im sorry i missed that... done now tho

File: src/components/Wiki/WikiAccordion/index.tsx [hunk 1]
```diff
@@ -6,22 +6,28 @@ import {
...
   BoxProps,
+  useBreakpointValue,
 } from '@chakra-ui/react'
 import { RiArrowDownSLine } from 'react-icons/ri'
 
 interface AccordionProps {
   title: string
   withNoDarkBg?: boolean
+  base: boolean
+  xl: boolean
```
Comment (kesar) [inline src/components/Wiki/WikiAccordion/index.tsx:17]: this naming should change. Nobody will know without checking implementation what's the meaning of xl or base.

to follow chakra style of props we could send an object of breakpoints and define if we want them collapsed or not. make it optional and default to non-collapsed if prop is not defined.

```
      <WikiAccordion title="Media" collapsed={{base: false, xl: false}}> ...
```

**Review by kesar — CHANGES_REQUESTED** [review]

**Review by kesar — APPROVED** [review]

**Pull request merged**